packages/react/src/components/Slider/Slider.js

```diff
@@ -140,11 +140,11 @@ export default class Slider extends PureComponent {
 
     /**
      * A value determining how much the value should increase/decrease by Shift+arrow keys,
-     * which will be `(max - min) / stepMuliplier`.
+     * which will be `(max - min) / stepMultiplier`.
      */
-    stepMuliplier: deprecate(
+    stepMultiplier: deprecate(
```

Comment:
This should not change. Changing the prop name would be breaking, that's why it's been deprecated in favor of a different prop. Please revert these changes.